common/src/main/java/com/wynntils/models/items/annotators/game/GearAnnotator.java

```diff
@@ -16,8 +16,8 @@
 import net.minecraft.world.item.ItemStack;
 
 public final class GearAnnotator implements GameItemAnnotator {
-    private static final Pattern GEAR_PATTERN =
```

Comment:
Maybe add a regexp test?

Reply:
My gradle setup is slightly broken and I can't run tests, and I have yet to fix that...

Reply:
Oh, that's really bad. :( I think this is a case where a test really would be beneficial, especially seeing as how this has been going back and forth with different strings that failed in one way or another.

I had a similar problem with a dev environment that I had customized yet were broken in some way. What I did was just clone the repo to another dir and open as a separate project in IntelliJ. I recommend you do that, and just run the test.

Reply:
I just pushed the tests, and if they fail, we will see it on the github workflow.